### PR TITLE
Show no action on email if user has any

### PR DESCRIPTION
**Pull request by molecula451**
##### Description

Ongoing testing & sendgrid..

##### Refers/Fixes

#6116 

##### Testing

debugmail

### CODE REVIEW

**Comment by molecula451** [comment]
@thelostone-mc Can you please take a look at the commits? I'm having trouble getting the delay bounty e-mail on debugging

**Comment by molecula451** [comment]
@thelostone-mc  Can you please take a look at this draft?

**Comment by molecula451** [comment]
@owocki this was suppose to be simple, sometime simple blows mind. I'm passing a false statement so it won't ship daily actions when there's 'not' mind checking ?

**Comment by owocki** [comment]
@molecula451 i dont understand the question. where are you passing false in?  mind commetning inline on the code so i can see contextually what were talking about..

File: app/marketing/mails.py [hunk 1]
```diff
@@ -915,6 +915,8 @@ def new_bounty_daily(bounties, old_bounties, to_emails=None):
         has_offer = is_email_townsquare_enabled(to_emails[0]) and is_there_an_action_available()
         if has_offer:
             offers = f"💰 1 New Action"
+        if not has_offer:
```
Comment (molecula451) [inline app/marketing/mails.py]: @owocki false statement

**Comment by molecula451** [comment]
closing this in favor of #6327

**Pull request closed without merging**